packages/core-js/utils/ast/finders.js

```diff
@@ -82,19 +84,30 @@ function findYulCaseValues(contractName, ast) {
   return items;
 }
 
-function findFunctionSelectors(contractName, asts) {
+function _findFunctionSelectors(contractNode) {
```

Comment:
Instead of making the single contract case private, could we also expose it and use a name for each that reflects the difference?